content/news/lpw-stepdown.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2020-07-22T00:00:00+00:00"
+title: "Lee-Ping Wang stepping down from the leadership role"
```

Comment:
from "a" leadership role, or from "his" leadership role?

Reply:
Fixed to "a".